### PR TITLE
feat(VolumeViewport3D): add volumeViewport3Dd sample hp

### PR DESCRIPTION
**Pull request by jbocce**
### Context

See issue #3195

### Changes & Results

- importing and using Cornerstone 3D's VolumeViewport3D viewport
- added mprAnd3DVolumeViewport hanging protocol
- allowed for W/L preset to be applied to a viewport

### Testing

Launch OHIF with the following query string `?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095722.1&hangingProtocolId=mprAnd3DVolumeViewport`

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] "OS: Windows 11"
- [x] "Node version: 18.10.0
- [x] "Browser: Chrome Version 110.0.5481.178"
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by jbocce** [comment]
@sedghi, there is one scenario that is not quite right IMHO...

1. With the 3D volume viewport selected/active, change the active to tool to something else (e.g. Length tool, pan, etc.)
2. The `TrackballRotateTool` is lost and we cannot get it back.

Possible fixes for this...
1. Add a new track ball UI tool in the top toolbar
2. It would be great if the trackball could simply always be the active tool in that volume viewport. I was able to obtain this by commenting out lines 410-413 and lines 417-420 in Cornerstone's ToolGroup.ts. However, I am not certain of the consequences of that.

Any other suggestions? That said perhaps the rest of the PR changes need an update that will change everything... 🤣I am open to anything. 😊

**Comment by sedghi** [comment]
Maybe I'm doing something wrong but this link doesn't like to show the correct thing

image

image

**Comment by jbocce** [comment]
> Maybe I'm doing something wrong but this link doesn't like to show the correct thing
> 
> https://deploy-preview-3241--ohif-platform-viewer.netlify.app/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095722.1&hangingProtocolId=mprAnd3DVolumeViewport
> 
> image

Nope... I did something wrong 🙁I updated the dependencies... hopeful that should fix things

File: extensions/cornerstone/src/services/ViewportService/Viewport.ts [hunk 1]
```diff
@@ -190,10 +191,10 @@ class ViewportInfo {
     }
 
     // map SAGITTAL, AXIAL, CORONAL orientation to be used by cornerstone
-    if (viewportOptionsEntry.viewportType?.toLowerCase() === VOLUME) {
-      orientation = getCornerstoneOrientation(viewportOptionsEntry.orientation);
-    } else {
+    if (viewportOptionsEntry.viewportType?.toLowerCase() === STACK) {
       orientation = Enums.OrientationAxis.AXIAL;
```
Comment (sedghi) [inline extensions/cornerstone/src/services/ViewportService/Viewport.ts:195]: there is no need to provide orientation for stackViewport

**Review by sedghi — APPROVED** [review]
Looks great thanks!

**Pull request merged**